### ISSUE TITLE
Incorrect `<<` behavior with strings

### ISSUE THREAD

**Issue by ike709** [wixoaGit/OpenDream#216]
Code:
```
/obj/item/weapon/circuitboard/vendor/proc/set_type(typepath, mob/user)
		build_path = typepath
		name = "circuit board ([names_paths[build_path]] Vendor)"
		user << "<span class='notice'>You set the board to Getmore Chocolate Corp.</span>"
```
Normally this should output the message to the `user` mob. But because the proc is called in `New()` and there's no sanity checking, `user` is null. Instead it causes:
```
Exception Occured: Invalid bit shift left operation on DreamValue(DreamObject, null) and DreamValue(String, "<span class='notice'>You set the board to Getmore Chocolate Corp.</span>")
=DM StackTrace=
   /obj/item/weapon/circuitboard/vendor/set_type(...)
   /obj/machinery/vending/snack/New(...)
   <InitDreamObject DreamObject(/obj/machinery/vending/snack)>


=C# StackTrace=
System.Exception: Invalid bit shift left operation on DreamValue(DreamObject, null) and DreamValue(String, "<span class='notice'>You set the board to Getmore Chocolate Corp.</span>")
   at OpenDreamRuntime.Procs.DMOpcodeHandlers.BitShiftLeft(DMProcState state) in C:\Users\main\Documents\GitHub\OpenDream\OpenDream\OpenDreamRuntime\Procs\DMOpcodeHandlers.cs:line 635
   at OpenDreamRuntime.Procs.DMProcState.InternalResume() in C:\Users\main\Documents\GitHub\OpenDream\OpenDream\OpenDreamRuntime\Procs\DMProc.cs:line 179
   at OpenDreamRuntime.ProcState.Resume() in C:\Users\main\Documents\GitHub\OpenDream\OpenDream\OpenDreamRuntime\DreamThread.cs:line 76
   ```

**Comment by ike709** [comment]
Note that `user` is null here.

**Comment by ike709** [comment]
For reference, in BYOND this silently fails with no runtime.

**Comment by wixoaGit** [comment]
Fixed in f2b6bcea087d8d6daffbe4a523d6f8271ab4726c.
